packages/tasks/src/model-libraries.ts

```diff
@@ -446,6 +446,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.pyannote_audio,
 		filter: true,
 	},
+	"py-feat": {
+		prettyLabel: "py-feat",
+		repoName: "py-feat",
+		repoUrl: "https://github.com/cosanlab/py-feat",
+		docsUrl: "https://py-feat.org/",
+		snippets: snippets.py-feat,
```

Comment:
This should be a code snippet that is adapted to the current model the user is visiting in the Hub, telling them how to use it in the library when they click on the "Use this Model" button.